www/m/js/modules/disasm/Disassembly.js

```diff
@@ -625,6 +632,7 @@ export class Disassembly extends RadareInfiniteBlock {
 			// { name: 'define flag color (fc)', shortcut: 'C', fct: function(evt, offset) { return applyOp(offset, 'C'); } },
 			// { name: 'set as data', shortcut: 'd', fct: function(evt, offset) { return applyOp(offset, 'd'); } },
 			// { name: 'end of function', shortcut: 'e', fct: function(evt, offset) { return applyOp(offset, 'e'); } },
+			{ aoj: false, name: 'view graph', shortcut: 'G', fct: function loadgraph() { return uiContext.navigateTo(Widgets.DISASSEMBLY_GRAPH);}},
```

Comment:
Anonymous functions is better here, or even arrow functions: `() => uiContext.navigateTo(...)`. No need to return.

You can refactor this part if you want.